src/systems/filecoin_mining/storage_mining/storage_miner_actor_code.go

```diff
@@ -175,10 +177,10 @@ func (a *StorageMinerActorCode_I) _onMissedSurprisePoSt(rt Runtime) {
 // construct PowerReport from SectorTable
 // need lastPoSt to search for new expired deals in _updateSectorUtilization since the lastPost
 // where DealExpirationAMT takes in a range of Epoch and return a list of values that expire in that range
-func (a *StorageMinerActorCode_I) _submitPowerReport(rt Runtime, lastPoStResponse block.ChainEpoch) {
+func (a *StorageMinerActorCode_I) _submitPowerReport(rt Runtime, lastPoStResponse block.ChainEpoch, processDealPayment bool) {
 	h, st := a.State(rt)
 	newExpiredDealIDs := st._updateSectorUtilization(rt, lastPoStResponse)
-	activePower := st._getActivePower(rt)
+	activePower, _ := st._getActivePower(rt, processDealPayment)
```

Comment:
I don't understand why you send it across here but don't harvest the return...?

Reply:
we implicitly use this in serialization, this is _ so that compiler doesnt complain.